Core/Content/Commands/FunCommands/ConvertTempCommand.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Discord;
+using Discord.Commands;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using TomatBot.Core.Content.Embeds;
+using TomatBot.Core.Framework.CommandFramework;
+using TomatBot.Core.Framework.DataStructures;
+
+namespace TomatBot.Core.Content.Commands.FunCommands
+{
+    public sealed class ConvertTempCommand : TomatCommand
+    {
+        public override MethodInfo? AssociatedMethod => GetType().GetMethod("HandleCommand");
+
+        public override HelpCommandData HelpData => new("convert", "Converts specified number from celsius to farenheit, or vice versa.");
+
+        public override CommandType CType => CommandType.Fun;
+
+        public override string Parameters => "<celsius/farenheit> <temperature>";
```

Comment:
Would be cleaner and prettier to simply have a parameter for the temperature unit, and have to convert the number from F->C and C->F, displaying both results.